doc/api/constants.md

```diff
@@ -0,0 +1,1083 @@
+# Constants
+
+The `constants` module provides convenient access to commonly used constants for
+use with file access, crypto methods, error handling and more. No methods are
+exposed by the `constants` module itself.
+
+**Note:** Specific constants may not be available on every operating system.
+
+## Error Constants
+
+The following error constants are adapted directly from the Posix `errno`
+standards. See http://man7.org/linux/man-pages/man3/errno.3.html for more
```

Comment:
… You are *fast* addressing nits. :smile:

The docs usually stylize it as `POSIX`, and manpage references like `errno(3)` would automatically be converted to the man7.org link.

And btw, for some of these errors, there’s already information in errno.md… not sure what to do about that (or if there’s something to be done about it), but some kind of cross-referencing might be nice.